internal/cli/atlas/maintenance/clear.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package maintenance
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type ClearOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store store.MaintenanceWindowClearer
+}
+
+func (opts *ClearOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+var clearTemplate = "Maintenance window removed.\n"
+
+func (opts *ClearOpts) Run() error {
+	err := opts.store.ClearMaintenanceWindow(opts.ConfigProjectID())
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(clearTemplate)
+}
+
+// mongocli atlas maintenanceWindow(s) clear [--projectId projectId]
+func ClearBuilder() *cobra.Command {
+	opts := &ClearOpts{}
+	cmd := &cobra.Command{
+		Use:     "clear",
+		Short:   clearMaintenanceWindow,
+		Aliases: []string{"rm", "delete"},
+		PreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
[q] should we confirm? I don't have a strong opinion right now but wanted to throw it out

Reply:
yes I think we should, I am going to change it